### DISCUSSION TITLE
Need explanation about filament Load Unload

### DISCUSSION THREAD

**Discussion by Chrisyahya** [Jyers/Marlin#1334] [Q&A] [answered]
SO i looked on wiki, it doesnt explained so well what it do.

Anyone know how much unload filament distance is? because i tried it, it didnt unload all the way to extruder. only unloads a bit (old FW, havent tried new version unload feature)

also how to use "load filament" function? i tried it, it didnt push to hotend, only few centimeters after extruder (old fw, havent tried the new one)

maybe need explanation like this in wiki:
unload filament - Push back your filament xx centimeter / unload completely out from extruder
Load filament - push filament to hotend / push xx centimeter to towards hotend

thank you!

**Comment by Cal-LF** [comment, marked as answer]
My settings are 455mm unload and 450mm load. Your settings will be different due to the length of the feeder tube.
Try the V1.3.4 firmware for your board and setup. The load and unload filament uses the length settings.

**Reply by Chrisyahya** [reply]
i tried it (havent changed values) but odd.. i tried unload feature, it does reverse back, and then push again by itself to the hotend, leaving excess melted plastic to the bed. does it supposed to? i thought its just reverse back then stops.

**Comment by asychev** [comment]
Sad that those options does not work reliably if you have BMG style extruder (too slow load, scratchy unload). @Jyers is it possible to do something in this case?

**Reply by ledcomp** [reply]
My BMG loads and unloads reliably but it is a little slow.

**Reply by asychev** [reply]
> Just to change the value in menu Control -> Max Speed -> Extruder : to 60 (or 80)
> and after don't forget to do a Save Settings

@tititopher68-dev It is already 60 by default. Should it be 180 because 3:1 ratio of BMG extruder?

**Reply by asychev** [reply]
BTW, it is not possible to set it > 50 from UI

**Reply by Jyers** [reply]
@asychev Compiling with a greater unload/load speed in Configuration_adv.h will help with this
